README.md

```diff
@@ -56,6 +56,8 @@ serverless deploy
 
 **5.** After completion of the step 4, navigate to the root of your project directory and run:
 ```bash
+# bundle the code using webpack
```

Comment:
lets create a new step and add this instead of doing it in the same `serverless deploy` command

Reply:
Addressed this remark